WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsActivity.kt

```diff
@@ -0,0 +1,74 @@
+package org.wordpress.android.ui.bloggingprompts
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.BloggingPromptsListActivityBinding
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.LocaleAwareActivity
+import org.wordpress.android.util.AppLog
+
+class BloggingPromptsActivity : LocaleAwareActivity() {
+    private lateinit var site: SiteModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(BloggingPromptsListActivityBinding.inflate(layoutInflater).root)
+
+        site = if (savedInstanceState == null) {
+            checkNotNull(intent.getSerializableExtra(WordPress.SITE) as? SiteModel) {
+                "SiteModel cannot be null, check the PendingIntent starting BloggingPromptsActivity"
+            }
+        } else {
+            savedInstanceState.getSerializable(WordPress.SITE) as SiteModel
+        }
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        if (item.itemId == android.R.id.home) {
+            onBackPressed()
+            return true
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    override fun onNewIntent(intent: Intent) {
```

Comment:
Considering we have `standard` launch mode for  `BloggingPromptsActivity`, I was wondering if the `onNewIntent` method would ever be called 🤔

Reply:
True. I'll remove this method.

Reply:
Done.